src/completers/completeReducer/test.js

```diff
@@ -0,0 +1,80 @@
+import Immutable from 'seamless-immutable';
```

Comment:
If we are using jest, I'd follow the framework convention to create test files.
This should be
`src/completers/__tests__/completeReducer`